AlicerceTests/ServiceLocatorTests.swift

```diff
@@ -0,0 +1,695 @@
+//
+//  ServiceLocatorTests.swift
+//  Alicerce
+//
+//  Created by Luís Afonso on 26/01/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import XCTest
+
+@testable import Alicerce
+
+class ServiceLocatorTests : XCTestCase {
+    
+    var serviceLocator = ServiceLocator()
```

Comment:
This is only a minor thing, but with `setUp` running before each test, do you think we need to create an object here? Normally I opt for an implicitly unwrapped optional `var serviceLocator: ServiceLocator!`.

What do you think?